lib/commands.py

```diff
@@ -647,8 +647,7 @@ def callback(x):
                 util.print_error('Got Response for %s' % address)
             except BaseException as e:
                 util.print_error(str(e))
-        h = self.network.addr_to_scripthash(address)
-        self.network.send([('blockchain.scripthash.subscribe', [h])], callback)
+        self.network.subscribe_to_addresses([addr], callback)
```

Comment:
should be `address` not `addr`

Reply:
will fix today

Reply:
Ah. You already did. :) Thanks for catching that!